velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -141,6 +142,17 @@ Writer::Writer(
   } else {
     flushPolicy_ = std::make_unique<DefaultFlushPolicy>();
   }
+
+  if (options.schema) {
+    ArrowSchema arrowSchema;
+    auto leafPool = pool_->addLeafChild("parquet-write-schema-convert");
+    exportToArrow(
+        BaseVector::create(options.schema, 0, leafPool.get()), arrowSchema);
+    PARQUET_ASSIGN_OR_THROW(
+        arrowContext_->schema, arrow::ImportSchema(&arrowSchema));
+    ArrowSchemaRelease(&arrowSchema);
```

Comment:
exportToArrow sets a custom `releaseArrowSchema` in Bridge.cpp.
We can avoid this explicit call. 
Remove the new include as well.

Reply:
Done.